docs/guides/main.rst

```diff
@@ -9,14 +9,34 @@ collection better.
 Installing
 ----------
 
-You will need Python.
-Beets works on Python 3.8 or later.
+Beets requires Python 3.9 or later, you will need to install that first. Depending
+on your operating system, you may also be able to install beets from a package
+manager, or you can install it with `pip`_.
 
-* **macOS** 11 (Big Sur) includes Python 3.8 out of the box.
-  You can opt for a more recent Python installing it via `Homebrew`_
-  (``brew install python3``).
-  There's also a `MacPorts`_ port. Run ``port install beets`` or
-  ``port install beets-full`` to include many third-party plugins.
+Using pip
+^^^^^^^^^
+
+To use the most recent version of beets, we recommend installing it with
+`pip`_, the Python package manager. If you don't have `pip`_ installed, you can
+follow the instructions on the `pip installation page`_ to get it set up.
+
+.. code-block:: console
```

Comment:
I'm pretty sure we push people towards using `pipx` instead, I think that's in `CONTRIBUTING.rst`

Reply:
Indeed, didn't know we had that. I changed it to pipx.